src/com/amplitude/api/TrackingOptions.java

```diff
@@ -0,0 +1,193 @@
+package com.amplitude.api;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class TrackingOptions {
+
+    private static String[] SERVER_SIDE_PROPERTIES = {
+        Constants.AMP_TRACKING_OPTION_CITY,
+        Constants.AMP_TRACKING_OPTION_COUNTRY,
+        Constants.AMP_TRACKING_OPTION_DMA,
+        Constants.AMP_TRACKING_OPTION_IP_ADDRESS,
+        Constants.AMP_TRACKING_OPTION_LAT_LON,
+        Constants.AMP_TRACKING_OPTION_REGION,
+    };
+
+    public static final String TAG = "com.amplitude.api.TrackingOptions";
+
+    Set<String> disabledFields = new HashSet<String>();
```

Comment:
Maybe these should be enums instead of strings? I don't think we use the string form ever, but it's not that big of a deal either way.